index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+        <h1 id="title">Survey form</h1>
+        <p id="description">Please fill this form</p>
+        <form id="survey-form">
+           <div> <label id="name-label" for="name">Name</label>
+            <input id="name" placeholder="name" type="text" required>
+        </div>
+
+            
+        <div>
+            <label id="email-label" for="email">Email</label>
+            <input id="email" Placeholder="email" type="email" required>
+        </div>
+            
+        <div><label id="number-label" for="number">Number</label>
+            <input id="number" type="number" placeholder="number" min="10" max="99" required>
+        </div>
+
+            <div>
+                <label id="dropdown-label" for="drowpdown"></label>
+            <select id="dropdown"></select>
+            </div>
+                <div>
+            <p>Would you recommend Us to a friend?</p>
+            <label>
+                <input
+                  name="user-recommend"
+                  value="definitely"
+                  type="radio"
+                  class="input-radio"
+                  checked
+                />Definitely</label>
```

Comment:
I would recommend that for better practice instead of using <input> as a child element of <label> you can use them as siblings and give them the same id!